docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Elytron_OIDC_Client.adoc

```diff
@@ -430,7 +430,7 @@ Here is an example of an `oidc.json` configuration file:
 
 * `logout-path`: The path component of a URL.  It is uses to identify the URL that is used to start an OIDC logout request.
 * `logout-callback-path`: An absolute URI.  It will be called after logout has been executed.  It is used by Front-channel and Back-channel logout.  The URI must be registered with the OpenId Provider.
-* `post-logout-uri` : An absolute URI.  It will be called after logout has been executed.  It is used by RP-Initiated logout only.
+* `post-logout-redirect-uri` : An absolute URI.  It will be called after logout has been executed.  It is used by RP-Initiated logout only.
```

Comment:
When front channel logout or back channel logout is also configured and a post-logout-redirect-uri has been set, we still get redirected to the post logout redirect uri right? If so, we should update this text.

Reply:
No for FrontChannel and BackChannel logout the property used is "logout-callback-path".  The property 
"post-logout-redirect-uri" is specific to RP-Initiated logout.

Reply:
Right but it's possible to use RP-initiated logout in combination with front channel logout or back channel logout.

Maybe we can just update the text to "It is used by RP-Initiated logout".

Reply:
After a lengthly discussion with Pedro that concept is not accurate.  Front-channel and Back-channel logout are built on top of RP-initiated logout functionality.  RP-initiated logout performs the getAuthServerUrl lookup function that;  Front and Back Channel use this.  

RP-initiated logout  can be used on it's own.  It will contact the OP to perform the logout.  If the post-logout-redirect-uri is declared the implementing app can leave the user on that endpoint else no further interaction is performed.